.circleci/config.yml

```diff
@@ -529,7 +531,10 @@ jobs:
                 --deb-db-bucket grafana-testing-aptly-db --deb-repo-bucket grafana-testing-repo --packages-bucket \
                 grafana-downloads-test --rpm-repo-bucket grafana-testing-repo --simulate-release
             else
-              /tmp/grabpl publish-packages --edition << parameters.edition >>
+              # TODO: Revert to non-simulation
+              /tmp/grabpl publish-packages --edition << parameters.edition >> \
+                --deb-db-bucket grafana-testing-aptly-db --deb-repo-bucket grafana-testing-repo --packages-bucket \
+                grafana-downloads-test --rpm-repo-bucket grafana-testing-repo --simulate-release
```

Comment:
Does `--simulate-release` mean the packages will be uploaded but not deploy to the repositories, or what will happen?

Reply:
@xlson You can study how the `--simulate-release` flag works [here](https://github.com/grafana/build-pipeline/blob/master/pkg/subcmd/publishpackages/action.go#L195).

Basically it will upload packages to GCS (although to alternate buckets, since CircleCI configures it as such) and not publish to grafana.com.

Reply:
Okay, sounds good.